autogpt_platform/backend/test/data/test_models.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+from pydantic import SecretStr
+
+from backend.data.model import HostScopedCredentials
+
+
+class TestHostScopedCredentials:
```

Comment:
pls put the tests in the module level like in all other test files

Reply:
Gosh, I've just opened a can of worms:

1. I didn't know people started adding tests outside the test folder. Not a fan of the style, but I'll go with it.
2. It defeats my conscience to update the move this file in the model and let the other live in the test folder, so I moved everything to each respective module.
3. No tests executed :/
4. Turns out we've been running tests only on the test folder on the CI.
5. Execute the test outside the test folder https://github.com/Significant-Gravitas/AutoGPT/pull/10215/files#diff-f9fe1667b472650b9522f94c8da9290e9167637dd2959f60111707e97a804699R193
6. Lots of them are broken </3 
7. So here we are now, having a bigger PR with some of those tests fixed, re-enabled, and moved on each module